build/dependencies.props

```diff
@@ -179,9 +184,9 @@
     <RemotionLinqPackageVersion>2.2.0</RemotionLinqPackageVersion>
     <SerilogExtensionsLoggingPackageVersion>1.4.0</SerilogExtensionsLoggingPackageVersion>
     <SerilogSinksFilePackageVersion>3.2.0</SerilogSinksFilePackageVersion>
-    <SQLitePCLRawBundleGreenPackageVersion>1.1.8</SQLitePCLRawBundleGreenPackageVersion>
-    <SQLitePCLRawBundleSqlcipherPackageVersion>1.1.8</SQLitePCLRawBundleSqlcipherPackageVersion>
-    <SQLitePCLRawCorePackageVersion>1.1.8</SQLitePCLRawCorePackageVersion>
+    <SQLitePCLRawBundleGreenPackageVersion>1.1.10-pre20180223200113</SQLitePCLRawBundleGreenPackageVersion>
```

Comment:
Where is this version coming from? I don't see it on nuget.

Reply:
https://www.nuget.org/packages/SQLitePCLRaw.bundle_green/1.1.10-pre20180223200113 ?

Reply:
huh, not sure why i didn't see it before

Reply:
Still seems like a very strange version number. I'm assuming there will be a 1.1.10 stable at some point?

Reply:
Yes, Eric said he was planning to have 1.1.10 ready sometime in the next few weeks.